core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -2604,23 +2604,27 @@ private[spark] object Utils extends Logging {
    *   - serializer(such as KryoSerializer) supports relocation of serialized objects
    */
   def isPushBasedShuffleEnabled(conf: SparkConf): Boolean = {
-    val serializer = Utils.classForName(conf.get(SERIALIZER)).getConstructor(classOf[SparkConf])
-      .newInstance(conf).asInstanceOf[Serializer]
-    val canDoPushBasedShuffle =
-      conf.get(PUSH_BASED_SHUFFLE_ENABLED) &&
-        (conf.get(IS_TESTING).getOrElse(false) ||
-          (conf.get(SHUFFLE_SERVICE_ENABLED) &&
-            conf.get(SparkLauncher.SPARK_MASTER, null) == "yarn" &&
-            // TODO: [SPARK-36744] needs to support IO encryption for push-based shuffle
-            !conf.get(IO_ENCRYPTION_ENABLED) &&
-            serializer.supportsRelocationOfSerializedObjects))
-
-    if (!canDoPushBasedShuffle) {
-      logWarning("Push-based shuffle can only be enabled when the application is submitted" +
```

Comment:
It also missed two spaces in "submitted to" and "and relocation". Just fixed it as well.